prow/cluster/jobs/istio-releases/pipeline/istio-releases.pipeline.master.yaml

```diff
@@ -0,0 +1,296 @@
+    always_run: false
+    run_if_changed: "(daily|monthly)/test/.*"
+    optional: true
+    labels:
+      preset-service-account: "true"
+    spec:
+      containers:
+      - <<: *istio_container
+        command:
+        - entrypoint
+        - prow/daily-upgrade-tests-1.0.2.sh
+      nodeSelector:
+        testing: test-pool
+
+  - name: rp-unit-tests
+    branches: *branch_spec
+    always_run: false
+    run_if_changed: "(daily|monthly)/test/.*"
+    labels:
+      preset-service-account: "true"
+    spec:
+      containers:
+      - <<: *istio_container
+        command:
+        - entrypoint
+        - prow/daily-unit-tests.sh
+      nodeSelector:
+        testing: test-pool
+
+  - name: rp-istioctl-tests
+    branches: *branch_spec
+    always_run: false
+    run_if_changed: "(daily|monthly)/test/.*"
+    labels:
+      preset-service-account: "true"
+    spec:
+      containers:
+      - <<: *istio_container
+        command:
+        - entrypoint
+        - prow/istioctl-tests.sh
+      nodeSelector:
+        testing: test-pool
+
+  - name: rp-e2e-pilot-no_auth
+    branches: *branch_spec
+    always_run: false
+    run_if_changed: "(daily|monthly)/test/.*"
+    labels:
+      preset-service-account: "true"
+    spec:
+      containers:
+      - <<: *istio_container
+        command:
+        - entrypoint
+        - prow/daily-e2e-pilot-no_auth.sh
+      nodeSelector:
+        testing: test-pool
+
+  - name: rp-e2e-bookinfoTests
+    branches: *branch_spec
+    always_run: false
+    run_if_changed: "(daily|monthly)/test/.*"
+    labels:
+      preset-service-account: "true"
+    spec:
+      containers:
+      - <<: *istio_container
+        command:
+        - entrypoint
+        - prow/daily-e2e-pilot-no_auth.sh
+      nodeSelector:
+        testing: test-pool
+
+  - name: rp-e2e-simpleTests
+    branches: *branch_spec
+    always_run: false
+    run_if_changed: "(daily|monthly)/test/.*"
+    labels:
+      preset-service-account: "true"
+    spec:
+      containers:
+      - <<: *istio_container
+        command:
+        - entrypoint
+        - prow/daily-e2e-pilot-no_auth.sh
+      nodeSelector:
+        testing: test-pool
+
+  - name: rp-e2e-mixer-no_auth
+    branches: *branch_spec
+    always_run: false
+    run_if_changed: "(daily|monthly)/test/.*"
+    labels:
+      preset-service-account: "true"
+    spec:
+      containers:
+      - <<: *istio_container
+        command:
+        - entrypoint
+        - prow/daily-e2e-pilot-no_auth.sh
+      nodeSelector:
+        testing: test-pool
+
+  - name: rp-daily-e2e-dashboard
+    branches: *branch_spec
+    always_run: false
+    run_if_changed: "(daily|monthly)/test/.*"
+    labels:
+      preset-service-account: "true"
+    spec:
+      containers:
+      - <<: *istio_container
+        command:
+        - entrypoint
+        - prow/daily-e2e-pilot-no_auth.sh
+      nodeSelector:
+        testing: test-pool
+
+postsubmits:
+
+  istio-releases/daily-release:
```

Comment:
is this supposed to be istio-releases/pipeline?

Reply:
ouch thanks for catching this